views/layout.hbs

```diff
@@ -92,9 +62,9 @@
           <div class="content">
             <a href="#proposition-links" class="js-header-toggle menu">Menu</a>
             <nav id="proposition-menu">
-              <a href="#availability" id="proposition-name">IRC Bed Management</a>
+              <a href="{{{ homepageUrl }}}" id="proposition-name">{{ title }}</a>
               <ul id="proposition-links" class="">
-                <li><a href="#availability" class="active">Bed availability</a></li>
+                <li><a href="{{{ homepageUrl }}}" class="active">Bed availability</a></li>
                 <li><a href="#statistics">Stats</a></li>
```

Comment:
I've just toyed with the Gulp file, have added some variables to pass the URL and menu text to the template.